profiles/base/crd.yaml

```diff
@@ -2,17 +2,18 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   creationTimestamp: null
-  labels:
-    controller-tools.k8s.io: "1.0"
-  name: profiles.kubeflow.org
+  name: profiles.profile.kubeflow.org
```

Comment:
Lets change the name back to profiles.kubeflow org for now.

Reply:
https://github.com/kubeflow/kubeflow/pull/4299